Gemfile.jruby-3.1.lock.release

```diff
@@ -509,7 +510,7 @@ GEM
       logstash-filter-grok (>= 4.4.1)
       logstash-mixin-ecs_compatibility_support (~> 1.2)
       stud (>= 0.0.22, < 0.1.0)
-    logstash-input-tcp (6.4.6-java)
+    logstash-input-tcp (7.0.2-java)
```

Comment:
This has a major update - this will bring in breaking changes to SSL

Reply:
I don't why kicked in a patch 🤔